tokio/src/runtime/mod.rs

```diff
@@ -312,6 +312,15 @@ cfg_rt! {
         blocking_pool: BlockingPool,
     }
 
+    /// The flavor of the `Runtime`.
+    #[derive(Debug, PartialEq)]
+    pub enum RuntimeFlavor {
```

Comment:
Please add a link to `Handle::runtime_flavor`.

Reply:
Also added, but not sure if that's what you meant.